.dockerignore

```diff
@@ -1,5 +1,4 @@
 node_modules
-grassroots-backend/node_modules
-grassroots-backend/dist
-grassroots-frontend/node_modules
-grassroots-frontend/dist
+**/node_modules
+**/dist
+docker/data
```

Comment:
This keeps us from copying in the caddy certs, which ran into permissions issues.

.dockerignore only effects the COPY in the Dockerfile, not the bind mounts, so it still gets bound in correctly.